variables.tf

```diff
@@ -0,0 +1,18 @@
+variable "cloudflare_secret_id" {
+  description = "The secret ID for the Cloudflare API token stored in AWS Secrets Manager"
+  type        = string
+}
+
+variable "records_map" {
+  description = "A map of records split with subdomain and zone information"
+  type = map(object({
+    subdomain = string
+    zone      = string
+  }))
+}
+
+variable "create_validation_records" {
```

Comment:
I would change this to enable_validation as it actual does two things, creates dnd validation records and also validations them